IFC4x3/Properties/i/IsReinforced_1Qp9XSGSD30AXUnzycyqDf/Documentation.md

```diff
@@ -0,0 +1 @@
+To indicate whether the OCS foundation is reinforced or not.
```

Comment:
Propose documentation " True if the OCS foundation  is reinforced.

Reply:
I have changed the documentation. Commit : a1b0cae8ebd1da839393bbf5ab16f2d1f2f833c2